docs/topics/null-safety.md

```diff
@@ -143,6 +143,17 @@ is `null`, the assignment is skipped and the expression on the right is not eval
 person?.department?.head = managersPool.getManager()
 ```
 
+## Nullable receiver
+
+Functions could be extended to use a [nullable receiver](extensions.md#nullable-receiver).
+This way you can check for the null condition and have a default defined in one place. 
```

Comment:
“Check for the null condition” sounds odd to me. Maybe something like “Specify behaviour for potentially null values without the need to use null-checking logic at each call-site” or something like that. Otherwise, the explanation focuses too much on “what you can do in the body”, rather than the actual effect this has (changing the calling convention)

Reply:
that is more precise, yeah [dee3b18](https://github.com/JetBrains/kotlin-web-site/pull/3133/commits/dee3b18d426b6f05ae9851974e57c3c4b955e3e6)